printer.js

```diff
@@ -98,8 +99,8 @@ Printer.prototype.println = function(content){
  * @param  {[String]}  encoding [description]
  * @return printer instance
  */
-Printer.prototype.text = function(content, encoding){
-  return this.print(iconv.encode(content + _.EOL, encoding || 'GB18030'));
+Printer.prototype.text = function (content, encoding) {
+  return this.print(iconv.encode(content + _.EOL, encoding || 'iso-8859-1'));
```

Comment:
Changing the default encoding character setting is still being discussed. #64 #67 #69